zucchini-ui-frontend/src/main.less

```diff
@@ -147,3 +147,8 @@
   transition: opacity linear 100ms;
   opacity: 0;
 }
+
+.error-message-modal-dialog {
+  max-width: 100%;
+  display: table;
```

Comment:
I did not find an easier to display the full errorMessage content (without scrollbars, ...)